README.md

```diff
@@ -1,3 +1,157 @@
-# Procedural-Ray-Tracing
-Repository for our Procedural Ray Tracing project, made by Y3 students at Breda University of Applied Science (former NHTV)
-More information: [Wiki](https://teamwisp.github.io/)
+# Wisp Rendering Framework
+We are a group of students from [Breda University of Applied Sciences]([https://www.buas.nl](https://www.buas.nl/)) working on a rendering framework using cutting-edge real-time ray-tracing technology from NVIDIA.
+
+## Table of contents
+
+- [Showcase](#showcase)
+
+- [Product overview](#product-overview)
+- [Features](#features)
+- [Contributing](#contributing)
+  - [Setting up your workspace](#setting-up-your-workspace)
+  - [Development pipeline](#development-pipeline)
+  - [Coding standards](#coding-standards)
+  - [Documentation standards](#documentation-standards)
+  - [Code architecture](#code-architecture)
+- [More information](#more-information)
+
+## Showcase
+
+| ![](media/sun_temple.png)                     | ![](media/shadows_reflections_in_reflections.png) |
+| --------------------------------------------- | ------------------------------------------------- |
+| ![](media/futuristic_vault_maya_viewport.png) | ![](media/sun_temple_soft_shadows.png)            |
+
+## Product overview
+
+Wisp is an advanced real-time ray tracing renderer built for the demands of video game artists. Wisp integrates with [Maya](https://www.autodesk.com/products/maya/overview) to give the user a real-time in-viewport representation of the final product. The goal of this project is to eliminate long render times during modelling by giving artists a real-time high quality render of the scene in the viewport.
+
+## Features
+
+You can find a more detailed list [here](http://teamwisp.github.io/feature_list) which also separates the tasks into critical and non-critical.
+
+- Global Illumination.
+- Ambient Occlusion.
+- Reflections.
+- Shadows.
+- Physically Based Rendering.
+- Translucent & Transparency.
+- GPU Accelerated.
+- Realtime rendering.
+- Built for NVIDIA RTX.
+- Implemented into Maya/Blender’s viewport.
+- Dynamic LOD’s for non-selected / background objects.
+
+## Contributing
```

Comment:
I would remove most of this header and just link to the TDD and mention the most important pages. It doesn't make sense for such a niece thing to take up so much space imo.